benchmark_runner/main/main.py

```diff
@@ -9,157 +9,169 @@
 from benchmark_runner.common.clouds.Azure.azure_operations import AzureOperations
 from benchmark_runner.common.clouds.IBM.ibm_operations import IBMOperations
 from benchmark_runner.clusterbuster.clusterbuster_workloads import ClusterBusterWorkloads
-from benchmark_runner.common.oc.oc import OC
 
 # logger
 log_level = os.environ.get('log_level', 'INFO').upper()
 logger.setLevel(level=log_level)
 
-# venv
-# python -m venv venv
-# . venv/bin/activate
 
 SYSTEM_EXIT_BENCHMARK_FAILED = 1
 SYSTEM_EXIT_UNKNOWN_EXECUTION_TYPE = 2
 
+benchmark_operator_workload = None
+benchmark_runner_workload = None
+clusterbuster_workload = None
```

Comment:
This really belongs somewhere below main; main shouldn't really know the details of what workloads are available.

Reply:
I put it above the main, not inside the main because it serve the method outside the main.
The problem before this fix was that all the methods where inside the main.